pkgs/development/compilers/edk2/default.nix

```diff
@@ -61,6 +61,7 @@ edk2 = stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pythonEnv ];
   depsBuildBuild = [ buildPackages.stdenv.cc buildPackages.util-linux buildPackages.bash ];
+  buildInputs = [ libuuid ];
```

Comment:
libuuid is aliased to util-linuxMinimal.
Do we actually need both util-linux going into depsBuildBuild and libuuid here?

Reply:
@LunNova during cross build it fail on missing `-luuid` (see linked issue https://github.com/NixOS/nixpkgs/issues/245188)